community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -65,6 +65,67 @@ class QueryPlanTest extends DocumentingTestBase {
     )
   }
 
+  @Test def constraintOperation() {
+    profileQuery(
+      title = "Constraint Operation",
+      text =
+        """Creates a constraint on a (label,property) pair.
+          |The following query will create a unique constraint on the `name` property of nodes with the `Country` label.
+          |This will ensure that we won't end up with duplicate `Country` nodes in our database.
+        """.stripMargin,
+      queryText = """CREATE CONSTRAINT ON (c:Country) ASSERT c.name is UNIQUE""",
+      assertion = (p) => assertThat(p.executionPlanDescription().toString, containsString("ConstraintOperation"))
+    )
+  }
+
+  @Test def distinct() {
+    profileQuery(
+      title = "Distinct",
+      text =
+        """Removes duplicate rows.
+          |The following query will return unique locations that have people working in them
+        """.stripMargin,
+      queryText = """MATCH (l:Location)<-[:WORKS_IN]-(p:Person) RETURN DISTINCT l""",
+      assertion = (p) => assertThat(p.executionPlanDescription().toString, containsString("Distinct"))
+    )
+  }
+
+  @Test def eagerAggregation() {
+    profileQuery(
+      title = "Eager Aggregation",
+      text =
+        """Eagerly loads resulting sub graphs before starting to emit the aggregated results.
+          |The following query will collect the people who work in every location before returning any rows.
+        """.stripMargin,
+      queryText = """MATCH (l:Location)<-[:WORKS_IN]-(p:Person) RETURN l.name AS location, COLLECT(p.name) AS people""",
+      assertion = (p) => assertThat(p.executionPlanDescription().toString, containsString("EagerAggregation"))
+    )
+  }
+
+  @Test def updateGraph() {
```

Comment:
Another way to do it would be to just list those operators somewhere, with a general description saying that they belong to the legacy planner. Then they get mentioned, and can be found, but it's rather little work.